Makefile

```diff
@@ -3,8 +3,8 @@
 all: test 
 
 test:
-	nosetests joblib
+	py.test joblib
```

Comment:
`--pyargs` interprets the argument as installed python library. Previously appveyor did that, hile Travis ran the tests in cloned repository itself, if I understood correctly.

On a separate note, I reverted a few commits to debug, I'll restore all of them back in a few minutes.